README.md

```diff
@@ -1,49 +1,44 @@
-# Setup
+# Shopify Admin, app extensions - Starter code
 
-## CLI
-If you have grabbed this template from the Shopify CLI you should be good to go!
+This repo contains starter code for apps that use _Argo_ to extend the UI of the _Shopify Admin_.       
 
-Start developing your awesome extension.
+For more about _Argo_ and admin extensions, please [read our docs](https://shopify.dev/tutorials/product-subscription-extension-overview).
 
+## Getting started
 
-## From github
-If you grabbed this template directly from github:
+[Use the Shopify App CLI to create your _Argo_-enabled extension](https://shopify.dev/tutorials/getting-started-product-subscription-extension#scaffold-a-product-subscription-app-extension).
 
-#### npm
-```bash
-## With npm
-npm install
+**Note:** There is no need for you to clone this repo directly.
+The CLI will take care of cloning the repo and generating all required files.
 
-npm run generate -- --type=PRODUCT_SUBSCRIPTION
+## Contributing
```

Comment:
Not sure about this one, do we expect 3p contributions here?

We also seem to have lost the 'Getting started' heading.